.travis.yml

```diff
@@ -2,7 +2,7 @@ language: julia
 
 julia:
   - 1.0
-  - 1.4
+  - 1.5
```

Comment:
```suggestion
  - 1
```
this shorthand will auto update when the next minor release is changed

Reply:
A good point - thank you!